central/graphql/resolvers/node_scan_benchmark_test.go

```diff
@@ -0,0 +1,176 @@
+package resolvers
```

Comment:
Can you post some benchmark results here?

Reply:
Do I add it in comment here? Or in the top of file?

Reply:
If you could include it in the PR description